lib/subscriber_list_mover.rb

```diff
@@ -69,12 +69,12 @@ def email_change_to_subscribers(source_subscriber_list)
 
     bulk_move_template = <<~BODY.freeze
       Hello,
-
-      You've subscribed to get email alerts about #{list_title} information.
-
-      GOV.UK is changing the type of email alerts you get to make sure you find out when any #{list_title} guidance is added or updated.
-      You can [change how often you get updates or unsubscribe](#{email_redirect}) from GOV.UK email alerts.
-
+      You've subscribed to get emails about #{list_title}.
```

Comment:
As this is parsed as markdown (well a rather restricted one) you may well find that, due to the lack of empty line, that this renders as `<p>Hello, You've subscribed to get emails about #{list_title}.</p>`

Reply:
Good spot thanks!